background.js

```diff
@@ -1,68 +1,76 @@
-"use strict";
+'use strict';
+
 let enabled = true;
 // Enable FPI when the extension runs
-chrome.privacy.websites.firstPartyIsolate.set({ value: true});
+chrome.privacy.websites.firstPartyIsolate.set({ value: true });
 
 function updateIcon(prefEnabled) {
-    // TODO maybe this needs a state of the enabled/disabled/error kind
-    // e.g., when levelOfControl == not_controllable
-    browser.browserAction.setIcon({
-        path: prefEnabled ?
-            "icons/fishbowl-light.svg":
-            "icons/fishbowl-disabled.svg"
-    });
-    browser.browserAction.setTitle({
-        // Screen readers can see the title
-        title: prefEnabled ?
-            browser.i18n.getMessage("TooltipDisableFPI") :
-            browser.i18n.getMessage("TooltipReEnableFPI"),
-    });
+  // TODO maybe this needs a state of the enabled/disabled/error kind
+  // e.g., when levelOfControl == not_controllable
+  browser.browserAction.setIcon({
+    path: prefEnabled ?
+      'icons/fishbowl-light.svg' :
+      'icons/fishbowl-disabled.svg'
+  });
+  browser.browserAction.setTitle({
+    // Screen readers can see the title
+    title: prefEnabled ?
+      browser.i18n.getMessage('TooltipDisableFPI') :
+      browser.i18n.getMessage('TooltipReEnableFPI')
+  });
 }
-/*function refreshUI() {
-    chrome.privacy.websites.firstPartyIsolate.get({},(details) => {
-            updateIcon(details.levelOfControl !== "not_controllable" ?
-                details.value :
-                false);
-    });
-}*/
-function setFPI(state) {
-        chrome.privacy.websites.firstPartyIsolate.set({ value: state }, () => {
-        if ((chrome.runtime.lastError === undefined) || (chrome.runtime.lastError === null)) {
-            enabled = state;
-        } else {
-            enabled = false;
-        }
-        updateIcon(enabled);
-    });
+
+/*
+function refreshUI() {
+  chrome.privacy.websites.firstPartyIsolate.get({},(details) => {
+    updateIcon(details.levelOfControl !== 'not_controllable' ?
+      details.value :
+      false);
+  });
 }
+*/
 
+function setFPI(state) {
+  chrome.privacy.websites.firstPartyIsolate.set({ value: state }, () => {
+    if ((typeof chrome.runtime.lastError === 'undefined') || (chrome.runtime.lastError === null)) {
```

Comment:
We could probably change this to `!chrome.runtime.lastError`

Reply:
Please do so :)